src/main/java/mate/academy/spring/service/dto/mapping/impl/response/OrderResponseMapper.java

```diff
@@ -0,0 +1,24 @@
+package mate.academy.spring.service.dto.mapping.impl.response;
+
+import java.util.stream.Collectors;
+import mate.academy.spring.model.Order;
+import mate.academy.spring.model.Ticket;
+import mate.academy.spring.model.dto.response.OrderResponseDto;
+import mate.academy.spring.service.dto.mapping.DtoResponseMapper;
+import org.springframework.stereotype.Component;
+
+@Component
+public class OrderResponseMapper implements DtoResponseMapper<OrderResponseDto,Order> {
+    @Override
+    public OrderResponseDto toDto(Order object) {
```

Comment:
```suggestion
    public OrderResponseDto toDto(Order order) {
```

Reply:
I'm not sure about "object" name in all of your Mapper classes, better to rename them